src/main/java/io/gravitee/repository/jdbc/ratelimit/JdbcRateLimitRepository.java

```diff
@@ -49,10 +50,13 @@
     @Autowired
     private JdbcTemplate jdbcTemplate;
 
+    @Value("${management.jdbc.prefix:}")
+    private String prefix;
+
     private final TransactionTemplate transactionTemplate;
 
-    private static String buildInsertStatement() {
-        return "insert into ratelimit (" +
+    private String buildInsertStatement() {
+        return "insert into " + getTableNameFor("ratelimit") + " (" +

```

Comment:
Add a constructor and initialize an instance variable with the table name and the prefix